packages/modular-scripts/src/__tests__/utils/__snapshots__/getWorkspaceInfo.test.ts.snap

```diff
@@ -12,6 +12,19 @@ exports[`getWorkspaceInfo 1`] = `
 `;
 
 exports[`getWorkspaceInfo 2`] = `
```

Comment:
This snapshot was updated due to material changes to the `packages` that exist in the project